src/protocol/resp/src/request/ping.rs

```diff
@@ -5,11 +5,15 @@
 use super::*;
 use std::io::{Error, ErrorKind};
 
-#[derive(Debug, PartialEq, Eq)]
-#[allow(clippy::redundant_allocation)]
-pub struct PingRequest {}
+#[metric(name = "ping2")]
```

Comment:
need to follow up w/ brian on this. There is metric conflict w/ pingserver blocking this. I dont think blocker for getting this merged can fix in follow up.